website/docs/index.html.markdown

```diff
@@ -177,6 +189,8 @@ The following arguments are used to configure the VMware NSX-T Provider:
 * `enforcement_point` - (Optional) Enforcement point, mostly relevant for policy
   data sources. For VMC environment, this should be set to `vmc-enforcementpoint`.
   For on-prem deployments, this setting should not be specified.
+* `policy_global_manager` - (Optional) True if this is a policy global manager
```

Comment:
Here too, just 'global_manager'. Also, can change the string 'policy global manager' to 'Global Manager'